src/AppBundle/Command/AddSingleUserCommand.php

```diff
@@ -137,8 +141,51 @@ protected function addSingleUser(OutputInterface $output, array $data, array $op
                 $ndr = new Ndr($client);
                 $em->persist($ndr);
             }
+        } else if (isset($data['caseNumber'])) {
+            // If client already exists, just assign user
+            $client = $clientRepo->findOneBy(['caseNumber' => CasRec::normaliseCaseNumber($data['caseNumber'])]);
+            if ($client instanceof Client) {
+                $user->addClient($client);
+            }
+        }
+
+        /**
+         * Prof/PA team members:
+         * Assign to team (creating team if necessary)
+        */
+        if (isset($data['teamName'])) {
+            $team = $teamRepo->findOneBy(['teamName' => $data['teamName']]);
+
+            if (!$team) {
+                $team = new Team($data['teamName']);
+                $em->persist($team);
+            }
+
+            $user->addTeam($team);
         }
 
+        /**
+         * Create report
+         */
+        $hierarchy = $this->getContainer()->getParameter('security.role_hierarchy.roles');
+        $roles = isset($hierarchy[$user->getRoleName()]) ? $hierarchy[$user->getRoleName()] : [$user->getRoleName()];
+        if ((in_array(User::ROLE_PROF, $roles) || in_array(User::ROLE_PA, $roles)) && isset($data['typeOfReport'])) {
+            try {
+                $type = CasRec::getTypeBasedOnTypeofRepAndCorref($data['typeOfReport'], $data['corref'], $user->getRoleName());
+            } catch (\Exception $e) {
+                if (in_array(User::ROLE_PROF, $roles)) {
+                    $type = $data['typeOfReport'] === 'OPG102' ? Report::TYPE_102_5 : Report::TYPE_103_5;
+                } else if (in_array(User::ROLE_PA, $roles)) {
+                    $type = $data['typeOfReport'] === 'OPG102' ? Report::TYPE_102_6 : Report::TYPE_103_6;
```

Comment:
Presume the fixtures dont account for 104-5 and 104-6?

Reply:
At the moment, no. We'll need to add it for DDPB-2524 though